src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -1,7 +1,88 @@
 package pl.byczazagroda.trackexpensesappbackend.controller;
 
-import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.ResultActions;
+import pl.byczazagroda.trackexpensesappbackend.BaseControllerTest;
+import pl.byczazagroda.trackexpensesappbackend.dto.CreateWalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.WalletDTO;
+import pl.byczazagroda.trackexpensesappbackend.service.WalletServiceImpl;
+
+import java.time.Instant;
+import java.util.Objects;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+class WalletControllerTest extends BaseControllerTest {
+
+    @Mock
+    private WalletServiceImpl walletService;
```

Comment:
We can use here WalletService type instead of WalletServiceImpl in order to follow dependency inversion principle

Reply:
Ok.